cmd/create/accountroles/cmd.go

```diff
@@ -333,19 +333,38 @@ func buildCommands(prefix string, permissionsBoundary string, accountID string,
 		if permissionsBoundary != "" {
 			permBoundaryFlag = fmt.Sprintf("\t--permissions-boundary %s \\\n", permissionsBoundary)
 		}
-		createRole := fmt.Sprintf("aws iam create-role \\\n"+
-			"\t--role-name %s \\\n"+
-			"\t--assume-role-policy-document file://sts_%s_trust_policy.json \\\n"+
-			"%s"+
-			"\t--tags %s"+
-			"\t--path %s",
-			name, file, permBoundaryFlag, iamTags, rolePath)
-		createPolicy := fmt.Sprintf("aws iam create-policy \\\n"+
-			"\t--policy-name %s \\\n"+
-			"\t--policy-document file://sts_%s_permission_policy.json"+
-			"\t--tags %s"+
-			"\t--path %s",
-			policyName, file, iamTags, policyPath)
+		createRole := ""
+		if rolePath == "" {
```

Comment:
NP
I was thinking of the following implementation:
```suggestion
		createRole := fmt.Sprintf("aws iam create-role \\\n"+
			"\t--role-name %s \\\n"+
			"\t--assume-role-policy-document file://sts_%s_trust_policy.json \\\n"+
			"%s"+
			"\t--tags %s"+
			name, file, permBoundaryFlag, iamTags)
		if rolePath != "" {
			createRole = fmt.Sprintf(createRole+"\t--path %s", rolePath)
		}
```
WDYT?

Reply:
> if rolePath != "" {
> 			createRole = fmt.Sprintf(createRole+"\t--path %s", rolePath)
> 		}

fixed